src/resmom/linux/mom_mach.c

```diff
@@ -129,6 +129,13 @@ static char	procfs[] = "/proc";
 static DIR	*pdir = NULL;
 static int	pagesize;
 static long	hz;
+
+/* convert between jiffies and seconds */
+#define	JTOS(x)	(((x) + (hz/2)) / hz)
```

Comment:
hz is set to sysconf(_SC_CLK_TCK) in mom_get_sample() only when it is zero. However, the static variable is not initialized above. Please initialize hz to zero.

Because hz is the denominator in the JTOS macro, we must be absolutely certain it is not set to zero when the macro is called. Please add an additional check in mom_get_sample() after hz has been set. Commenting here because GitHub won't let me add a line comment in the area I am referring to.